packages/react/src/common/accessibility.ts

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Currently this is only used in Grid, shouldn't we keep it there for the time being?

Reply:
https://gemeente-amsterdam.atlassian.net/browse/DES-1033